js/actions/index.ts

```diff
@@ -243,6 +266,16 @@ function watchCollection(db: firebase.firestore.Firestore, path: string, receive
     snapshot => snapshot.docs.map(doc => doc.data()));
 }
 
+function watchAnonymousAnswers(db: firebase.firestore.Firestore, source: string, runKey: string, dispatch: Dispatch) {
+// Setup Firebase observer. It will fire each time the resource structure is updated.
```

Comment:
This comment doesn't match what the function is doing. It will fire when the answers are updated.